### ISSUE TITLE
[TextAnalytics] Entity Negation property is not exposed in Text Analytics for Health client

### ISSUE THREAD

**Issue by shwars** [Azure/azure-sdk-for-python#16931]
- **Package Name**: azure.ai.textanalytics
- **Package Version**: 5.1.0b5
- **Operating System**: Windows
- **Python Version**: 3.7.4

**Describe the bug**
Text analytics for health service returns negation information for each entity (`isNegated`). This information is not exposed through Text Analytics Python SDK

**To Reproduce**
1. Install text analytics client: `pip install azure.ai.textanalytics==5.1.0b5`
2. Create the client:
```python
from azure.core.credentials import AzureKeyCredential
from azure.ai.textanalytics import TextAnalyticsClient

text_analytics_client = TextAnalyticsClient(
    endpoint=endpoint, 
    credential=AzureKeyCredential(key),
    api_version="v3.1-preview.3")
```
3. Call the service:
```python
documents = ["I have not been administered any aspirin, just 300 mg or favipiravir daily."]
poller = text_analytics_client.begin_analyze_healthcare_entities(documents, show_stats=True)
result = poller.result()
result = list(result)
```
4. Print out any entity in the text: `result[0].entities[1]`, or show all available properties `dir(result[0].entities[1])`.

Result:
```
HealthcareEntity(text=300 mg, category=Dosage, subcategory=None, length=6, offset=47, confidence_score=1.0, data_sources=None, related_entities={HealthcareEntity(text=favipiravir, category=MedicationName, subcategory=None, length=11, offset=57, confidence_score=1.0, data_sources=[HealthcareEntityDataSource(entity_id=C1138226, name=UMLS), HealthcareEntityDataSource(entity_id=J05AX27, name=ATC), HealthcareEntityDataSource(entity_id=DB12466, name=DRUGBANK), HealthcareEntityDataSource(entity_id=398131, name=MEDCIN), HealthcareEntityDataSource(entity_id=C462182, name=MSH), HealthcareEntityDataSource(entity_id=C81605, name=NCI), HealthcareEntityDataSource(entity_id=EW5GL2X7E0, name=NCI_FDA)], related_entities={}): 'DosageOfMedication'})
```
A property showing whether entity is negated is not available, eg. `result[0].entities[0].is_negated` does not work, and no similar properties are found in `dir` output.

**Expected behavior**
A property showing whether entity is negated should be available, eg. `result[0].entities[0].is_negated`.

**Comment by xiangyan99** [comment]
Thanks for the feedback, we’ll investigate asap.

**Comment by iscai-msft** [comment]
hey @shwars, support fo `is_negated` was not added in this release. For our next release, we will actually be expanding on this concept with `assertion` in our next release. For more information, you can see what an assertion looks like in [this](https://westus2.dev.cognitive.microsoft.com/docs/services/TextAnalytics-v3-1-preview-4/operations/HealthStatus) REST API example.

Closing since this is known and we already have plans to add this in our subsequent release. Thanks @shwars for reporting!

**Comment by iscai-msft** [comment]
This has been fixed in the `5.1.0b6` release @shwars, my apologies I should have updated you earlier. Thank you so much for your patience!

[Here](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-ai-textanalytics/5.1.0b6/azure.ai.textanalytics.html#azure.ai.textanalytics.HealthcareEntity) are the docs for this property in `HealthcareEntity`, thanks!
